frontend/src/components/BrushSelector.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div id="brush-selector">
+    <div id="color-selector"></div>
+    <BrushColorTile
+      v-for="color in colors"
+      :key="color"
+      :color="color"
+      @colorSelected="onColorSelected"
+    />
+    <div id="size-selector">
+      <BrushSizeTile v-for="size in sizes" :key="size" :size="size" @sizeSelected="onSizeSelected" />
+    </div>
+  </div>
+</template>
+
+<script>
+import BrushSizeTile from "./BrushSizeTile.vue";
+import BrushColorTile from "./BrushColorTile.vue";
+
+export default {
+  name: "BrushSelector",
+  components: {
+    BrushSizeTile,
+    BrushColorTile
+  },
+  props: {
+    colors: Array,
+    sizes: Array
+  },
+  data: function() {
+    return {
+      currentSize: this.sizes[0],
+      currentColor: this.colors[0]
+    };
+  },
+  methods: {
+    onSizeSelected: function(size) {
+      this.currentSize = size;
+      this.emitUpdatedBrush();
+    },
+    onColorSelected: function(color) {
```

Comment:
Take a look at how we use eventbus. 

https://www.digitalocean.com/community/tutorials/vuejs-global-event-bus

I think we can use this method of emitting and listening can be used for components. And stuff related to WebSocket events can be done using EventBus. What do you think Sam?

Reply:
I think it'd be beneficial for everything to be using EventBus. That way if we ever need other components listening in on this brush stuff (for whatever reason), they can just listen in on the correct "key" and it's gucci